conan-platforms/conan-profile-Darwin-x86_64.info

```diff
@@ -0,0 +1,12 @@
+[settings]
+os=Macos
+os_build=Macos
+arch=x86_64
+arch_build=x86_64
+compiler=apple-clang
+compiler.version=10.0
```

Comment:
Unfortunately, I can only use `9.0` for this. Trying to update my system to see if that will fix it.